### ISSUE TITLE
Cannot use keyboard to navigate through the info side panel for videos in Learn

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9175]
## Observed behavior
When the informational side panel is open for videos, a learner cannot navigate through the opened side panel.

|Informational side panel in video in Learn|
|--|
|<img width="860" alt="Screen Shot 2022-03-11 at 4 30 53 PM" src="https://user-images.githubusercontent.com/13563002/157995502-f1e0317f-664d-4db8-b844-e41b8531a661.png">|

## Errors and logs
None

## Expected behavior
Learners should be able to use their keyboard to navigate the side panel, as seen below:
![2022-03-11 16 26 47](https://user-images.githubusercontent.com/13563002/157995317-de80ee0a-ba3a-48be-b041-cbd16f333af4.gif)

## User-facing consequences
Accessibility issue

## Steps to reproduce
1. As learner, use keyboard to navigate to a video in a lesson
2. Use the keyboard to navigate to the `Info` icon and press `Return`
3. Notice that you cannot navigate through this side panel using the keyboard.

## Context
Kolibri 0.15
MacOS
Chrome

**Comment by marcellamaki** [comment]
Closed with https://github.com/learningequality/kolibri/pull/9408
